cypress/e2e/buckets.test.ts

```diff
@@ -394,5 +431,97 @@ describe('Buckets', () => {
       // mymeasurement comes from fixtures/data.txt
       cy.getByTestID('selector-list mymeasurement').should('exist')
     })
+
+    it('create scraper', () => {
```

Comment:
Scrapers are not supported for cloud.  We need to have this changed to support OSS only.   Check with one of the developers to find out how to turn this turn off for cloud v2 applications.

Reply:
Here's how to determine whether the app is running in cloud mode or not:
Check the following env variable `CLOUD_URL`,  if it's set to `\auth` then DO NOT run the scrapers tests (you are in the cloud environment).  Please make these changes to all scrapers tests.

Reply:
@drdelambre is it possible to detect if `CLOUD_URL` has that value from the cypress tests? If not, what do we do instead?

Reply:
not from the cypress tests. we'd have to run an oss version of the cluster, then run the tests against it. there's been informal talk around having "cloud only", "oss only", and "shared" subsets of the test that we'd run in their respective environments, but nothing has been formalized / spec'd to coincide with our GA strategy. once all that's done, then we can introduce that concept into the test runner.